Readme.md

```diff
@@ -1,76 +1,169 @@
+- Reusable workflows: they include the jobs that will be reused by other repositories. These jobs use Github actions to perform the required tasks inside each of the pipelines: integration, delivery and deployment. All the workflows are available inside the `.github/workflows` folder. 
+-  Caller templates: templates that references the corresponding reusable workflows. All the templates are available inside the `caller-workflow-templates` folder. To add the desired functionality to your repository, you should include the appropriate templates in the `.github/workflows` folder of your repository. For more information see [Add caller workflows](#add-caller-workflows)
 
-The workflows are divided based on the 3 different stages inside the CI/CD pipeline: continuous integration, continuous delivery and continuous deployment. Moreover, each one of these stages also contains workflows for the different distribution/delivery strategies that Graasp repositories use: AWS S3 (or S3 apps) or AWS ECS.
+The workflows are classified based on the 3 different stages inside the CI/CD pipeline: Continuous Integration, Continuous Delivery and Continuous Deployment. Moreover, each one of these stages also contains workflows for the different distribution/delivery strategies that Graasp repositories use: AWS S3 (or S3 apps) and AWS ECS.
 
 ## Features
 ### Continuous Integration
 Process within development that continuously builds, tests and merges new code changes. CI allows developers to deploy to the development environment. There, developers can see how new features will work and test improvements without affecting real users. 
 
-This functionality is provided by workflows with the syntax `cintegration-*.yml`. When triggered, three jobs are run: test, build and deploy
+This functionality is provided by workflows with the syntax `cintegration-*.yml`. When triggered, three jobs are run: test, build and deploy. Use this workflow family whenever you want to automate the deployment of your repository to the development environment. 
+
+### Update saging versions
+When the code is considered to be production-ready, it can be promoted to the staging environment.
+
+This functionality is provided by the `update-staging-versions.yml` workflow. Use this workflow whenever you want to automate the addition of new tags from your repository to the ecosystem staging stack in graasp-deploy. 
 
 ### Continuous Delivery
-Automated process that takes the validated code additions from the CI process and relases them to the stage environment. This isolated environment is as similar to the production environment as it can be, and it is where all of the hard core testing will be performed. 
+Automated process that takes the validated code additions from the previous process and relases them to the staging environment. This isolated environment is as similar to the production environment as it can be, and it is where all of the hard core testing will be performed. 
```

Comment:
```suggestion
Automated process that takes the validated code additions from the previous process and releases them to the staging environment. This isolated environment is as similar to the production environment as it can be, and it is where all of the hard core testing will be performed.
```